APIBacklog/documentation/APIBacklog.md

```diff
@@ -3,51 +3,61 @@
- | IMEI Fraud | MTN | 2023/08/13| [Link](https://github.com/camaraproject/WorkingGroups/pull/277) | MTN, GSMA Open Gateway, Huawei | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
- | WebRTC | Telefónica | 2023/08/16| [Link]([https://github.com/camaraproject/WorkingGroups/pull/280](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/SupportingDocuments/API%20proposals/APIproposal_WebRTC_Telefonica.md)) | Telefónica, Mavenir, Orange, Vodafone, TMUS | YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
-  | Network Insights | Verizon | 2023/08/31| [Link](https://github.com/camaraproject/WorkingGroups/pull/295) | Verizon, 5GFF, GSMA Open Gateway| YYYY/MM/DD | Under Evaluation | N/A | N/A | N/A | TBD | TBD | TBD | TBD |
+
+
+
+## API backlog
+
+ This is a live doc that captures the status of all the APIs which have been formally registered in CAMARA project. 
+
+ | **API [family] name**  |  **API [family] owner** |  **API [family] proposal registration date**  | **Link to the filled-in application template**   |  **Supporters in API Backlog Working Group**  |  **TSC decision (date)** |  **Sub Project**  |  **API name**  |  **API documentation** |  **API version**  | **API definition**  |  **Implementation**  |  **API tested at** |  **API deployed at company and country**  |  
+ | --- |  --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |---| --- |
+ | Quality on Demand | DTAG | N/A | N/A | DTAG, Orange, Telefonica, Vodafone|  Approved (N/A) | [QualityOnDemand](https://github.com/camaraproject/QualityOnDemand) | qod | [Link](https://github.com/camaraproject/QualityOnDemand/tree/main/documentation/API_documentation) | v0.9.0 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.9.0/code/API_definitions) | [PI_2 (Orange)](https://github.com/camaraproject/QualityOnDemand_PI2) | DTAG |  N/A |
+ |  | | | | |  | | |  | v0.8.1 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.1/code/API_definitions) | | | DTAG | DTAG - Germany |
+|  | | |   |  | | |  | v0.8.0 | [YAML](https://github.com/camaraproject/QualityOnDemand/tree/release-0.8.0/code/API_definitions) | | [PI_1 (DT)](https://github.com/camaraproject/QualityOnDemand_PI1)<BR>[PI_3 (SFN)](https://github.com/camaraproject/QualityOnDemand_PI3)| DTAG | DTAG - Germany |<!---Simple Edge Discovery --->
+| Simple Edge Discovery | Vodafone | N/A | N/A  | 5GFF, Vodafone | Approved (N/A)  | [EdgeCloud/Discovery](https://github.com/camaraproject/EdgeCloud/tree/main/code/API_definitions/Discovery) | simple-edge-discovery | [Link](https://github.com/camaraproject/EdgeCloud/tree/main/documentation) | v0.9.1 | [YAML](https://github.com/camaraproject/EdgeCloud/blob/main/code/API_definitions/Discovery/simple_edge_discovery.yaml) | N/A | N/A | N/A| N/A|<!---MEC Management --->
```

Comment:
The link is pointing to main, not to a release branch or tagged commit. 
There is a tagged version v0.8.1 https://github.com/camaraproject/EdgeCloud/blob/v0.8.1/code/API_definitions/simple_edge_discovery.yaml